From 2cdefbe920c12da70295c227ac8292b00f8be710 Mon Sep 17 00:00:00 2001
Message-Id: <patch-4.6-2cdefbe920c-20210510T104653Z-avarab@gmail.com>
In-Reply-To: <cover-0.6-00000000000-20210510T104653Z-avarab@gmail.com>
References: <cover-0.4-00000000000-20210505T121857Z-avarab@gmail.com>
	<cover-0.6-00000000000-20210510T104653Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 6 Apr 2021 16:24:13 +0200
Subject: [PATCH v2 4/6] Makefile: split up the deceleration of PERL_DEFINES
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Split the declaration of PERL_DEFINES across multiple line, making
this easier to read.

In 07d90eadb50 (Makefile: add Perl runtime prefix support, 2018-04-10)
when PERL_DEFINES was added only the RUNTIME_PREFIX was put on its own
line the rest weren't formatted like that for consistency. Let's do
that to make this consistent with most of the rest of this Makefile.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3ed6828de67..4f68f5e1dba 100644
--- a/Makefile
+++ b/Makefile
@@ -2270,7 +2270,10 @@ perl_localedir_SQ = $(localedir_SQ)
 
 ifndef NO_PERL
 PERL_HEADER_TEMPLATE = perl/header_templates/fixed_prefix.template.pl
-PERL_DEFINES = $(PERL_PATH_SQ) $(PERLLIB_EXTRA_SQ) $(perllibdir_SQ)
+PERL_DEFINES =
+PERL_DEFINES += $(PERL_PATH_SQ)
+PERL_DEFINES += $(PERLLIB_EXTRA_SQ)
+PERL_DEFINES += $(perllibdir_SQ)
 PERL_DEFINES += $(RUNTIME_PREFIX)
 
 # Support Perl runtime prefix. In this mode, a different header is installed
-- 
2.31.1.838.g924d365b763

